watchfaces/001-analog-70s-classic.qml

```diff
@@ -0,0 +1,349 @@
+
+    Canvas {
+        z: 1
+        id: minuteHand
+        property var minute: 0
+        property var rotM: (minute - 15)/60
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.reset()
+            ctx.lineCap="round"
+            ctx.beginPath()
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.8)
+            ctx.shadowOffsetX = 1
+            ctx.shadowOffsetY = 1
+            ctx.shadowBlur = 3
+            ctx.lineWidth = parent.width*0.034
+            ctx.strokeStyle = Qt.rgba(1, 1, 1, 1)
+            //circle in center
+            ctx.arc(parent.width/2, parent.height/2, parent.height*0.014, 0, 2*Math.PI, false)
+            ctx.moveTo(parent.width/2,
+                       parent.height/2)
+            //outer line
+            ctx.lineTo(parent.width/2+Math.cos(rotM * 2 * Math.PI)*width*0.327,
+                    parent.height/2+Math.sin(rotM * 2 * Math.PI)*width*0.327)
+            ctx.stroke()
+            ctx.closePath()
+            ctx.lineWidth = parent.width*0.015
+            ctx.strokeStyle = Qt.rgba(0, 0, 0, 1)
+            ctx.beginPath()
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0)
+            ctx.shadowOffsetX = 0
+            ctx.shadowOffsetY = 0
+            ctx.shadowBlur = 0
+            //inner line
+            ctx.moveTo(parent.width/2+Math.cos(rotM * 2 * Math.PI)*width*0.17,
+                       parent.height/2+Math.sin(rotM * 2 * Math.PI)*width*0.17)
+            ctx.lineTo(parent.width/2+Math.cos(rotM * 2 * Math.PI)*width*0.324,
+                    parent.height/2+Math.sin(rotM * 2 * Math.PI)*width*0.324)
+            ctx.stroke()
+            ctx.closePath()
+        }
+    }
+
+    Canvas {
+        z: 2
+        id: secondHand
+        property var second: 0
+        anchors.fill: parent
+        smooth: true
+        onPaint: {
+            var ctx = getContext("2d")
+            ctx.reset()
+            ctx.shadowColor = Qt.rgba(0, 0, 0, 0.7)
+            ctx.shadowOffsetX = 1
+            ctx.shadowOffsetY = 1
+            ctx.shadowBlur = 2
+            ctx.strokeStyle = "red"
+            ctx.lineWidth = parent.height*0.008
+            ctx.beginPath()
+            ctx.moveTo(parent.width/2, parent.height/2)
+            ctx.lineTo(parent.width/2+Math.cos((second - 45)/60 * 2 * Math.PI)*width*0.1,
+                    parent.height/2+Math.sin((second - 45)/60 * 2 * Math.PI)*width*0.1)
+            ctx.stroke()
+            ctx.closePath()
+            ctx.beginPath()
+            ctx.fillStyle = "red"
+            ctx.arc(parent.width/2, parent.height/2, parent.height*0.012, 0, 2*Math.PI, false)
+            ctx.fill()
+            ctx.moveTo(parent.width/2, parent.height/2)
+            ctx.lineTo(parent.width/2+Math.cos((second - 15)/60 * 2 * Math.PI)*width*0.32,
+                    parent.height/2+Math.sin((second - 15)/60 * 2 * Math.PI)*width*0.32)
+            ctx.stroke()
+            ctx.closePath()
+
```

Comment:
You should beware of those empty lines at the begining and end of blocks that don't actually carry information on the structure of the code ;) they are misleading when reading your code.